src/components/LeftMenu/index.jsx

```diff
@@ -0,0 +1,91 @@
+import { ReactComponent as Dashboard } from 'assets/icons/dashboard.svg';
+import { ReactComponent as List } from 'assets/icons/list.svg';
+import { ReactComponent as Logout } from 'assets/icons/logout.svg';
+import { ReactComponent as Menu } from 'assets/icons/menu.svg';
+import { ReactComponent as Saved } from 'assets/icons/saved.svg';
+import { ReactComponent as Settings } from 'assets/icons/settings.svg';
+import { ReactComponent as Transaction } from 'assets/icons/transaction.svg';
+import { ReactComponent as UserProfile } from 'assets/icons/user-profile.svg';
+import { ReactComponent as UserRole } from 'assets/icons/user-roles.svg';
+import React, { useState } from 'react';
+import {
+  LinkTitle,
+  MenuAnchor,
+  MenuItemInnerContainer,
+  MenuItemsContainer,
+} from './leftMenu.styles';
+
+/**
+ * Left side menu.
+ *
+ * @return {JSX.Element}
+ */
+function LeftMenu() {
+  const [animationType, setAnimationType] = useState('');
```

Comment:
Lets try to control the state of the Nav menu to become user friendly. the thinking process of giving a component it's state should be as of a user trying to use that component, not as a developer 
```suggestion
  const [collapsed, setIsCollapsed] = useState('');
```